bittensor/subtensor.py

```diff
@@ -3351,62 +3349,150 @@ def tempo(self, netuid: int, block: Optional[int] = None) -> Optional[int]:
     def get_total_stake_for_hotkey(
         self, ss58_address: str, block: Optional[int] = None
     ) -> Optional["Balance"]:
-        """Returns the total stake held on a hotkey including delegative"""
+        """
+        Returns the total stake held on a hotkey including delegative.
+
+        Args:
+            ss58_address (str): The SS58 address of the hotkey.
+            block (Optional[int], optional): The block number to retrieve the stake from. If ``None``, the latest
+                block is used. Default is ``None``.
+
+        Returns:
+            Optional[Balance]: The total stake held on the hotkey, or ``None`` if the hotkey does not
+                exist or the stake is not found.
+        """
         _result = self.query_subtensor("TotalHotkeyStake", block, [ss58_address])
-        if not hasattr(_result, "value") or _result is None:
-            return None
-        return Balance.from_rao(_result.value)
+        return (
+            None
+            if _result is None or not hasattr(_result, "value")
+            else Balance.from_rao(_result.value)
+        )
 
     def get_total_stake_for_coldkey(
         self, ss58_address: str, block: Optional[int] = None
     ) -> Optional["Balance"]:
-        """Returns the total stake held on a coldkey across all hotkeys including delegates"""
+        """
```

Comment:
what is the reasoning behind reducing these comments and not having args and return vars?

Reply:
Below this docstring describes all the arguments and the return value. 
Or did I misunderstand your question?